app/models/manageiq/providers/openstack/cloud_manager/refresher.rb

```diff
@@ -1,75 +1,14 @@
 module ManageIQ::Providers
-  class Openstack::CloudManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
-    include ::EmsRefresh::Refreshers::EmsRefresherMixin
-
-    def collect_inventory_for_targets(ems, targets)
-      targets_with_data = targets.collect do |target|
-        target_name = target.try(:name) || target.try(:event_type)
-
-        _log.info("Filtering inventory for #{target.class} [#{target_name}] id: [#{target.id}]...")
-        if ::Settings.ems.ems_refresh.openstack.try(:inventory_object_refresh)
-          inventory = ManageIQ::Providers::Openstack::Builder.build_inventory(ems, target)
-        end
-
-        _log.info("Filtering inventory...Complete")
-        [target, inventory]
-      end
-
-      targets_with_data
-    end
-
-    def parse_legacy_inventory(ems)
-      ManageIQ::Providers::Openstack::CloudManager::RefreshParser.ems_inv_to_hashes(ems, refresher_options)
-    end
-
+  class Openstack::CloudManager::Refresher < ManageIQ::Providers::BaseManager::ManagerRefresher
     def save_inventory(ems, target, inventory_collections)
       super
       EmsRefresh.queue_refresh(ems.network_manager) if target.kind_of?(ManageIQ::Providers::BaseManager)
       EmsRefresh.queue_refresh(ems.cinder_manager) if target.kind_of?(ManageIQ::Providers::BaseManager)
```

Comment:
so maybe, we will have to do a cinder full refresh always here, we should track that as a separate issue though